Sources/tart/Commands/Create.swift

```diff
@@ -23,14 +23,16 @@ struct Create: AsyncParsableCommand {
 
   func run() async throws {
     do {
-      let vmDir = try VMStorageLocal().create(name)
+      let tmpVMDir = try VMDirectory.temporary()
```

Comment:
will it be cleaned up in case of a Ctrl+C?

Reply:
Now it will be: https://github.com/cirruslabs/tart/pull/109/commits/4037f7ffb1e562c38920f0e12c8ab5e3355126d1.